tools/template.xml.in

```diff
@@ -1,113 +1,120 @@
+    <interface type="user">
+      <mac address="52:54:00:92:d4:7b" />
+      <model type="e1000-82545em" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x02" function="0x0" />
     </interface>
-    <serial type='pty'>
-      <target type='isa-serial' port='0'>
-        <model name='isa-serial'/>
+    <serial type="pty">
+      <target type="isa-serial" port="0">
+        <model name="isa-serial" />
       </target>
     </serial>
-    <console type='pty'>
-      <target type='serial' port='0'/>
+    <console type="pty">
+      <target type="serial" port="0" />
     </console>
-    <input type='mouse' bus='usb'>
-      <address type='usb' bus='0' port='1'/>
+    <input type="tablet" bus="usb">
+      <address type="usb" bus="0" port="1" />
     </input>
-    <input type='keyboard' bus='usb'>
-      <address type='usb' bus='0' port='2'/>
+    <input type="keyboard" bus="usb">
+      <address type="usb" bus="0" port="2" />
     </input>
-    <graphics type='spice' autoport='yes'>
-      <listen type='address'/>
-      <image compression='off'/>
+    <graphics type="spice">
+      <listen type="none" />
+      <image compression="off" />
+      <gl enable="no" />
     </graphics>
-    <sound model='ich9'>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1b' function='0x0'/>
+    <sound model="ich9">
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1b" function="0x0" />
     </sound>
     <video>
-      <model type='qxl' ram='65536' vram='65536' vgamem='16384' heads='1' primary='yes'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x01' function='0x0'/>
+      <model type="vga" vram="131072" heads="1" primary="yes" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x01" function="0x0" />
     </video>
-    <redirdev bus='usb' type='spicevmc'>
-      <address type='usb' bus='0' port='3'/>
+    <redirdev bus="usb" type="spicevmc">
+      <address type="usb" bus="0" port="3" />
     </redirdev>
-    <redirdev bus='usb' type='spicevmc'>
-      <address type='usb' bus='0' port='4'/>
+    <redirdev bus="usb" type="spicevmc">
+      <address type="usb" bus="0" port="4" />
     </redirdev>
-    <memballoon model='none'/>
+    <memballoon model="none" />
   </devices>
+  <seclabel type="dynamic" model="selinux" relabel="yes" />
   <qemu:commandline>
-    <qemu:arg value='-cpu'/>
-    <qemu:arg value='Penryn,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,+pcid,+ssse3,+sse4.2,+popcnt,+avx,+aes,+xsave,+xsaveopt,check'/>
-    <qemu:arg value='-device'/>
-    <qemu:arg value='isa-applesmc,osk=ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc'/>
-    <qemu:arg value='-smbios'/>
-    <qemu:arg value='type=2'/>
+    <qemu:arg value="-cpu" />
+    <qemu:arg value="Penryn,kvm=on,vendor=GenuineIntel,+invtsc,vmware-cpuid-freq=on,vmx=on,+pcid,+ssse3,+sse4.2,+popcnt,+avx,+aes,+xsave,+xsaveopt,check" />
```

Comment:
What's the implication of adding the `vmx=on` parameter? Is it needed? My understanding is that is is used to have KVM guests run other KVM virtual machines ([source](https://www.linux-kvm.org/page/Nested_Guests) and [source](https://www.kernel.org/doc/Documentation/virtual/kvm/nested-vmx.txt)).